frontend/src/app/types/app/menu-item.ts

```diff
@@ -3,4 +3,5 @@ export default interface MenuItem {
   name: string;
   icon: string;
   route?: string;
+  roles?: string;
```

Comment:
This is a great idea, but instead of a string, you'll want this to be an array, this way you can can do a better check for a role

Reply:
Updated the roles to be a string array. Marking as fixed.